test/config.yaml

```diff
@@ -1,4 +0,0 @@
-sampletable: pasilla_sampletable.tsv
-sample_dir: pasilla
-assembly: dm6
-data_dir: /data/LCDB/references
```

Comment:
sure, how about something like
```
resources/
  requirements.txt
  test_data/
    pasilla/
      get-data.sh
      sampletable.tsv
      config.yaml
      adapters.fa
    splicing/
      get-data.sh
      sampletable.tsv
      config.yaml    
```

and move the existing `schemas` over to `lcdb` maybe? Then I could build some tests for the schema config generation on top of what you currently have in that dir.